robottelo.properties.sample

```diff
@@ -303,6 +303,31 @@ ssh_key=
 # 'semanage port -a -t websm_port_t -p tcp <port-range>'
 port_range=9091, 14999
 
+# [ec2]
+# AWS EC2 to be added as a compute resource.
+# Access key ID of the IAM user
+# access_key=
```

Comment:
Some of these settings are not supposed to be taken from properties files directly as we don't have a shared account currently, but should be sourced from Ci . @kbidarkar What do you think?